api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -270,47 +270,26 @@ public static String getFileAsString(File file) throws IOException {
 	 * @param inputStream Stream to copy from
 	 * @param outputStream Stream/location to copy to
 	 * @throws IOException thrown if an error occurs during read/write
+	 * @return true if file copy was successful, false if was not
+	 * @should not copy the file when outputStream is null
+	 * @should not copy the file when inputStream is null
+	 * @should copy the file
```

Comment:
can you test that the outputstream was closed as well and you are not copying files although the method name says it (I know its misleading), but copying input to outputstream

I suggest tests like
"should close the outputstream if inputstream is null"
"should copy inputstream to outputstream and close the outputstream"